web/src/lib/utils/handle-error.ts

```diff
@@ -1,7 +1,12 @@
 import type { ApiError } from '@api';
+import { CanceledError } from 'axios';
 import { notificationController, NotificationType } from '../components/shared-components/notification/notification';
 
 export async function handleError(error: unknown, message: string) {
+  if (error instanceof CanceledError) {
```

Comment:
It seems we can now use `handleError` to handle exceptions in `getAssetsByBucket` (`asset.store.ts`)